pytorch_lightning/plugins/training_type/ddp_spawn.py

```diff
@@ -201,7 +202,10 @@ def new_process(self, process_idx: int, trainer: "pl.Trainer", mp_queue: SimpleQ
         if self.sync_batchnorm:
             self.model = self.configure_sync_batchnorm(self.model)
 
-        self.configure_ddp()
+        # skip wrapping the model if we are not fitting as no gradients need to be exchanged
```

Comment:
in L202-203, should we skip configuring sync batchnorm if we're in validate/test/predict too?

Reply:
Yeah, make sense, created a PR for it
https://github.com/PyTorchLightning/pytorch-lightning/issues/9243